www/src/utils/i18n.js

```diff
@@ -1,19 +1,59 @@
+const langs = require("../../i18n.json")
```

Comment:
only thing I could potentially NIT here is this - it's first time this is included in browser bundle, but I guess this will ultimately be needed anyway for language selector down the road (plus it's pretty small .json file for now). But we need this at least for codes here so this is fine.